frontend/packages/kubevirt-plugin/src/components/vm-status/vm-statuses.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { PodKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { getNamespace, getName } from '@console/shared';
+import { VM_STATUS_IMPORTING, VM_STATUS_IMPORT_ERROR } from '../../statuses/vm/constants';
+import { VMKind } from '../../types';
+import { getVMStatus } from '../../statuses/vm/vm';
+import { getVMImporterPods } from '../../selectors/pod/selectors';
+import { VmStatus } from './vm-status';
+
+const getId = (value) => `${getNamespace(value)}-${getName(value)}`;
+
+export const VmStatuses: React.FC<VmStatusesProps> = (props) => {
+  const { vm, pods, migrations } = props;
+  const statusDetail = getVMStatus(vm, pods, migrations);
+  const importerPods = getVMImporterPods(pods, vm);
+
+  switch (statusDetail.status) {
+    case VM_STATUS_IMPORTING:
+    case VM_STATUS_IMPORT_ERROR:
+      return (
+        <>
+          {importerPods.map((pod) => (
+            <div key={getId(pod)}>
+              <VmStatus {...props} pods={[pod]} verbose />
+            </div>
+          ))}
+        </>
+      );
+    default:
```

Comment:
Is this (empty) `default` clause needed, given the code below?

Reply:
done